Libs/MRML/Logic/vtkMRMLPlotLayoutLogic.h

```diff
@@ -0,0 +1,34 @@
+///
+///  vtkMRMLPlotLayoutLogic.cxx - slicer logic class for handling the observations
+///  of PlotLayout Nodes to Plot Nodes
+
+#ifndef __vtkMRMLPlotLayoutLogic_h
+#define __vtkMRMLPlotLayoutLogic_h
+
+// MRMLLogic includes
+#include "vtkMRMLAbstractLogic.h"
+
+
+class VTK_MRML_LOGIC_EXPORT vtkMRMLPlotLayoutLogic : public vtkMRMLAbstractLogic
```

Comment:
Reading this short introduction:  https://www.slicer.org/wiki/Documentation/Nightly/Developers/MRML#MRML_Node_References

As well as the more  exhaustive documentation could be useful: https://www.slicer.org/wiki/Documentation/Nightly/Developers/MRML/NodeReferences